src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+/*
+* sorts the addressbook
```

Comment:
Follow the style of the other commands (capitalise first letter + add full stop).